Makefile

```diff
@@ -15,4 +25,23 @@ update-charm-libs:
 check-dashboard-updates:
 	./scripts/check-dashboard-updates.sh
 
-.PHONY: help update-charm-libs check-dashboard-updates
+clean:
+	@echo "clean files"
+	@git clean -ffXd -e '!.idea' -e '!.vscode'
+	@echo "Cleaning existing build"
+	@rm -rf ${PROJECTPATH}/${CHARM_NAME}*.charm
+	@charmcraft clean
+	@echo "Remove download snap"
+	@rm -rf ${PROJECTPATH}/*.snap
+
+build: clean
+	@echo "Building charm"
+	@charmcraft -v pack
+
+download-snap:
+	wget -q https://github.com/canonical/openstack-exporter-operator/releases/download/rev2/golang-openstack-exporter_amd64.snap -O ./golang-openstack-exporter_amd64.snap
+
+integration: download-snap build
+	CHARM_LOCATION=${CHARM_LOCATION} CHARM_SNAP_LOCATION=${CHARM_SNAP_LOCATION} tox -e integration
```

Comment:
Missing `charm_name` env  see `local-charm-overlay.yaml.j2`

Reply:
`charm_name` env is assign by zaza.